src/ec/packages/ec-component-accordion2/accordion2.story.js

```diff
@@ -1,36 +1,49 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { buttonLabels, getExtraKnobs } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import demoData from './demo/data';
 
 import accordion2 from './ecl-accordion2.html.twig';
 import notes from './README.md';
 
+const prepareAccordion2 = data => {
+  data.items.forEach((item, index) => {
+    const { content, toggle, id } = item;
+
+    item.id = select(`item ${index} id`, [id], id, buttonLabels.required);
```

Comment:
- These labels are supposed to be the way in code you would retrieve a certain prop:

`item ${index} id` produces `item 0 id` which is not the way you would write the code to access the property, this is instead:
`items[0].id`
because your object is named items and it's an array, item is just a conventional name you used in the forEach, and you get this format by writing:
`items[${index}].id`

- The same is valid for all the others label you generated but:

- iconShape for sure is not something we pass to the template as such


- i don't see the icon_path knob and i don't see the "level" knob either, which were the changes requested in the previous review, @Joosthe so i'll leave also the previous "conversation" open.